Documentation/gettingstarted/egress-gateway.rst

```diff
@@ -0,0 +1,174 @@
+.. only:: not (epub or latex or html)
+
+    WARNING: You are looking at unreleased Cilium documentation.
+    Please use the official rendered version released here:
+    https://docs.cilium.io
+
+.. _egress_gateway:
+
+**********************
+Egress Gateway (beta)
+**********************
+
+Egress gateway provides a way to configure the egress policy. Without
+this feature, pod traffic could only egress from the node it is running
+on and masqueraded with the node IP.
+
+This document explains how to configure Cilium to enable egress gateway
+as well as how to configure egress NAT policy to route and SNAT the egress
+traffic for a specific workload.
+
+.. include:: ../beta.rst
+
+Configure Cilium
+================
+
+.. include:: k8s-install-download-release.rst
+
+The Cilium Egress Gateway feature relies on :ref:`kubeproxy-free`,
+follow the guide to create a new deployment. The beta feature is disabled by default.
```

Comment:
```suggestion
follow the guide to create a new deployment. The feature is disabled by default.
```
No need to remind that it is in beta mode here. Plus this is likely something we would forget to remove when moving out of beta.